http/templates/season_base.html.tera

```diff
@@ -1,29 +1,18 @@
 {% extends "base" %}
 
 {% block body %}
-<div class="bg-sky-600 h-7 mb-2 border-t">
-    <div class="flex text-indigo-900">
-        <div class="flex w-32">
-        </div>
-        <div class="flex flex-auto text-black" title="Season #{{season.id}}">
-            {{ season.format }}
-        </div>
-        <div class="flex flex-auto">
-            <a href="/season/{{season.id}}/brackets">Brackets</a>
-        </div>
-        <div class="flex flex-auto">
-            <a href="/season/{{season.id}}/standings">Standings</a>
-        </div>
-        <div class="flex flex-auto">
-            <a href="/season/{{season.id}}/qualifiers">Qualifiers</a>
-        </div>
-        <div class="flex md:w-1/5">
-        </div>
+<div id="season-container">
+    <div id="season-banner">
+        <div id="season-title">Season {{base_context.current_season.id}} - {{ season.format }}</div>
```

Comment:
this is wrong for 2 reasons:

1. it uses `current_season` but it needs to use the id of the season we're looking at
2. database id is not the correct ordinal of the season anyway (i didn't burn an id, so database id 1 is the second season of league; i try to just avoid using the id in the UI at all)

also for dumb historical reasons `format` is really more like `title` - the "format" of season 2 is just "Season 2"